src/app/loginComponents/accounts/external/revoke-token-dialog/revoke-token-dialog.component.html

```diff
@@ -0,0 +1,42 @@
+<!--
+    Copyright 2018 OICR
+
+    Licensed under the Apache License, Version 2.0 (the "License")
+    you may not use this file except in compliance with the License
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<h1 mat-dialog-title>Are you sure you want to revoke this Dockstore token?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>This action will be irreversible and you will need to log back in to generate a new Dockstore token.</p>
+  <mat-divider></mat-divider>
+  <div *ngIf="username" class="text-center">
+    <p>Please enter your account username to revoke.</p>
```

Comment:
Sort of torn on this one -- we don't log-in with an account username, so users may not know their username. I had to see what mine was when I tried your build locally :). In hindsight it's obvious, but hadn't remembered it was the same as my GitHub username. 

You can leave it, but I suggest:

1. Require a hard-coded phrase, e.g., "Revoke me"
2. Put the username in the message, e.g., "Please enter your account username, coverbeck, to revoke."